src/cargo/core/package.rs

```diff
@@ -167,6 +167,14 @@ impl Package {
     pub fn proc_macro(&self) -> bool {
         self.targets().iter().any(|target| target.proc_macro())
     }
+    // TODO fix this. For now, just wanted it to return a plausible value. Must figure out why .kinds() returns a Vec.
+    /// Gets crate-type in { .., artifact = <crate-type> } of this package
```

Comment:
This returns ArtifactKind of a package's _dependencies_, not the ArtifactKind of the package itself. This is why (bin/cdylib/staticlib) suffixes currently print on the wrong line.
```diff
- expected
+ actual
- foo v0.0.0 ([ROOT]/foo)
-└── bindep v0.0.0 (bin) ([ROOT]/foo/bindep)
+ foo v0.0.0 (bin) ([ROOT]/foo)
+ └── bindep v0.0.0 ([ROOT]/foo/bindep)
```